js/logic.js

```diff
@@ -292,7 +308,7 @@ const logic = (function () {
    * @return {boolean} success or fail
    */
   function pauseTimer(pause) {
-    if (!isRunning) return false;
+    if (!isRunning || isFlowing) return false;
```

Comment:
Would this lead to the timer not pausing during flow state? I would imagine that would be a nice to have if someone, e.g. got raided

Reply:
Tested it out, works well